maintainers/maintainer-list.nix

```diff
@@ -13977,4 +13977,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  nidabdella = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.